cmd/rpcdaemon/commands/trace_api_adhoc.go

```diff
@@ -0,0 +1,35 @@
+package commands
+
+import (
+	"context"
+)
+
+// Call Implements trace_call
+func (api *TraceAPIImpl) Call(ctx context.Context, req TraceFilterRequest) ([]interface{}, error) {
+	var stub []interface{}
```

Comment:
For the methods that are not implemented, I would prefer to return and error "Not implemented" instead of empty stub. It is better for the users to distinguish between "not implemented", and "works incorrectly"

Reply:
In the next PR...